docs/src/getting-started/scarb.md

```diff
@@ -9,6 +9,8 @@ Starknet Foundry uses [Scarb](https://docs.swmansion.com/scarb) to:
 
 One of the core concepts of Scarb is its [manifest file](https://docs.swmansion.com/scarb/docs/reference/manifest.html) - `Scarb.toml`.
 It can be also used to provide [configuration](../projects/configuration.md) for Starknet Foundry.
+Moreover, you can modify behaviour of `scarb test` to run `snforge` as 
+described [here](https://docs.swmansion.com/scarb/docs/extensions/testing.html#using-third-party-test-runners).
```

Comment:
Should it be a separate subsection on this page?

Reply:
I would leave it as it is